### PR TITLE
Breastfeeding intervention doc with preliminary data sources

### PR DESCRIPTION
**Pull request by alibow**

### CODE REVIEW

File: docs/source/intervention_models/breastfeeding_promotion/index.rst [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
...d young child feeding (IYCF) nutrition interventions on breasfeeding practices, growth, and mortality in low- and middle-income countries was recently perfored by [Lassi-et-al-2020]_. [Lassi-et-al-2020]_ reviewed several types of interventions related to IYCF practices, including interventions to promote a) early (within an hour of birth), b) exclusive breastfeeding (measured at three and six months), and b) continued breastfeeding (measured at 12, 18, and 24 months). Notably, of the 38 studies related to breastfeeding education interventions, none reported outcomes specific to continued breastfeeding. Based on six studies (very low quality evidence), breastfeeding interventions were associated with 2.02 (95% CI: 1.88, 2.17) times exclusive breastfeeding at three months; sensitivity analysis for allocation concealment removed five studies and yielded an RR=1.65 (95% CI: 1.49, 1.84). Based on 19 studies (very low quality evidence), breastfeeding education interventions were associated with 1.53 (95% CI: 1.47, 1.58) times the rate of exclusie breastfeeding at six months; sensitivity analaysis for allocation concealment removed five studies and yielded an RR=1.37 (95% CI: 1.32, 1.42).
```
Comment (plinds) [inline docs/source/intervention_models/breastfeeding_promotion/index.rst:28]: *performed in second line

Comment (plinds) [inline docs/source/intervention_models/breastfeeding_promotion/index.rst:28]: *exclusive in second to last line

File: docs/source/intervention_models/breastfeeding_promotion/index.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
... the National Nutrition Program set a target to increase the rate of exclusive breastfeeding by 22% between 2016 and 2020, although only a one percent increase was achieved by 2019. The current routine care practice regarding IYCF practices in Ethiopia includes perinatal care packages provided at health facilities for four focused antenatal visits, delivery care, and five contacts of postnatal care. As part of newborn care immediately after birth women are encouraged to initiate breastfeeding within one hour and are counselled on correct positioning. They are advised to exclusively breastfeed for the first six months at each postnatal contact. However, a barrier to these care services is low attendance of facility delivery rates and postnatal visits, particularly for rural mothers. Outreach programs for such women exist as part of the current community-based nutrition program in Ethiopia, which includes health extension workers and local women peer-educators, also known as the Women Development Army leaders; however, these workers are overburded with assigned tasks and are hindered in provided support that lactating mothers need, particularly during the immediate postpartum period.
```
Comment (plinds) [inline docs/source/intervention_models/breastfeeding_promotion/index.rst:30]: *overburdened in second to last line

File: docs/source/intervention_models/breastfeeding_promotion/index.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+The study conducted by [Abdulahi-et-al-2021]_ evaluated a community-based peer-led breastfeeding education and support intervention delivered during the prenatal and postnatal period through the established Women Development Army system in comparison to routine care. The study was conducted in Manna district located in Jimma Zone in southwest Ethiopia in rural setting. Allocation of study arms was conducted at the sub-district level among 36 sub-districts in the study location. Women were recruited during their second or third trimester or pregnancy. Peer-supporters made visits to women in the intervention arm twice in the last trimester of pregnancy (during the 8th and 9th month); on the 1st or 2nd, 6th or 7th, and 15th day after delivery; and thereafter monthly until the infant was five months old. Women in the control arm received routine care from health extension workers (described above).
```
Comment (plinds) [inline docs/source/intervention_models/breastfeeding_promotion/index.rst:32]: second or third trimester *of pregnancy (4th line)

File: docs/source/intervention_models/breastfeeding_promotion/index.rst [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+infant and young child feeding [were] available within the
+health/nutrition care system or through community outreach ... **to some degree**" (p 23). 
+
+Given the landscape of breastfeeding education and support in Ethiopia as described by [Abdulahi-et-al-2021]_ and the proposed changes to teh protocol in the intervention arm of this study, we will model **zero baseline coverage of the improved breastfeeding support intervention** (as the intervention arm in [Abdulahi-et-al-2021]_) and 100% coverage of routine care (control arm in [Abdulahi-et-al-2021]_) in Ethiopia and use the implied effect size associated with the transition from existing routine care to the improved intervention protocol from [Abdulahi-et-al-2021]_.
```
Comment (plinds) [inline docs/source/intervention_models/breastfeeding_promotion/index.rst:55]: changes to *the protocol

**Review by plinds — APPROVED** [review]

**Review by NathanielBlairStahn — APPROVED** [review]

**Pull request merged**